### ISSUE TITLE
Null nested models are instantiated (with null fields).

### ISSUE THREAD

**Issue by PtiPingouin** [gfx/Android-Orma#301]
Consider the following code, where Model2 is also a table:
```
@Table
public class Model1 {
    @Nullable
    protected Model2 model2;
}
```
When reading from database an instance of Model1 that has no model2, model1 still gets a model2 instance, with null values.

In Model1_Schema, I can see there are no condition for instantiating model2:

```
  @NonNull
  @Override
  public Model1 newModelFromCursor(@NonNull OrmaConnection conn, @NonNull Cursor cursor, int offset) {
    Model1 model = new Model1();
    model.setModel2(Model2_Schema.INSTANCE.newModelFromCursor(...));
    return model;
  }
```

Model2_Schema.INSTANCE.newModelFromCursor() is called no matter what.

Am I missing something?

**Comment by gfx** [comment]
What's your Orma version?

**Comment by PtiPingouin** [comment]
3.0.0-rc3

**Comment by gfx** [comment]
Seems bugs. Will investigate it.

**Comment by PtiPingouin** [comment]
Thanks

**Comment by gfx** [comment]
NOTE: This issue should be fixed in https://github.com/gfx/Android-Orma/pull/263 but the PR didn't solve it.

**Comment by gfx** [comment]
Released v3.0.0-rc4 with this bug fix. Can you try it please?

**Comment by PtiPingouin** [comment]
Ok
